js/mobile-menu.js

```diff
@@ -1,46 +1,223 @@
+/* eslint-disable func-names */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-shadow */
```

Comment:
why are we disabling Eslint rules...

Please provide valid use-cases where they are entirely unavoidable, otherwise lets remove all this and fix the issues instead. Please communicate on issues that you're unable to fix

Reply:
#### This comment is highly appreciated, issues on this comment as been checked and resolved.